helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterContext.java

```diff
@@ -105,6 +130,10 @@ public int getEstimatedMaxTopStateCount() {
     return _estimatedMaxTopStateCount;
   }
 
+  public float getEstimatedMaxUtilization() {
```

Comment:
Projected

Reply:
Same as what I replied previously, I think "estimate" is better.